contribution/lang/de.json

```diff
@@ -3278,7 +3278,7 @@
  	 	 	 	"eng": "AI Core Cluster"
  	 	 	},
  	 	 	"description": {
- 	 	 	 	"trans": "AI-Kerncluster enthält 1000 AI-Kerne. kann leicht zusammengebaut und demontiert werden.",
+ 	 	 	 	"trans": "AI-Kerncluster enthält 1000 AI-Kerne. Kann leicht zusammengebaut und demontiert werden.",
```

Comment:
Valid point. Corrected it, but changed it to 'KI-Kerncluster enthalten' or maybe 'Ein KI-Kerncluster enthält' might be better. Just 'KI-Kerncluster enthält' sounds incorrect, because KI-Kerncluster would be by nature plural in german I'd say without any numbering word attached to it.